timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -8,6 +8,10 @@ types:
       base-type: string
       external:
         java: com.palantir.atlasdb.timelock.api.Namespace
+    GenericNamedMinTimestamp:
```

Comment:
Is "Generic*" adding much here? It's a historical artifact that we're trying to generify the immutable timestamp, maybe in the future it will not exist, and then will stuck with the weird prefix

Reply:
Ah yeah, not much

I think I wanted to have `GenericNamedMinTimestamp` and `ReservedNamedMinTimestamp`

But, the latter is only immutable ts and so I just left it as a constant string

I'd make sense to convert it to one without prefix